src/util/__tests__/convertFieldWithFallback-typescript-test.ts

```diff
@@ -0,0 +1,36 @@
+/**
+ * Copyright (c) 2017-present, NDLA.
+ *
+ * This source code is licensed under the GPLv3 license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ */
+
+import { convertFieldWithFallback } from '../convertFieldWithFallback';
+
+const article = {
+  id: '2',
+  created: '2014-12-24T10:44:06Z',
+  title: { title: 'Tester', language: 'nb' },
+  tags: { tags: ['apekatt', 'eplekjekk'], language: 'nb' },
+};
+
+test('typescript: convertFieldWithFallback convert field title to string', () => {
+  const obj = convertFieldWithFallback<'title', string>(article, 'title', '');
+  expect(obj).toMatchSnapshot();
```

Comment:
Skjønner godt snapshots ved store returverdier, men her er snapshot 'Tester'. Hadde vore enklere å sjekke direkte. Eller er tanken å slippe å oppdaterte kode dersom ting skulle endre seg? Kun oppdatere snapshot?

Reply:
Er ikke det tanken ved alle snapshot tester?
Grunnen til at jeg gikk for snapshot her fremfor var egentlig bare fordi de andre testene (i js utgaven) var skrevet sånn.

> Hadde vore enklere å sjekke direkte.

Ser ikke helt forskjellen, men jeg kan godt endre den, men da skjønner jeg ikke hvorfor vi har snapshot-tester i det hele tatt :woozy_face:

Reply:
Er ikke det tanken ved alle snapshot tester?
Grunnen til at jeg gikk for snapshot her fremfor var egentlig bare fordi de andre testene (i js utgaven) var skrevet sånn.

> Hadde vore enklere å sjekke direkte.

Ser ikke helt forskjellen, men jeg kan godt endre den, men da skjønner jeg ikke hvorfor vi har snapshot-tester i det hele tatt :woozy_face:

Reply:
Er ikke det tanken ved alle snapshot tester?
Grunnen til at jeg gikk for snapshot her fremfor var egentlig bare fordi de andre testene (i js utgaven) var skrevet sånn.

> Hadde vore enklere å sjekke direkte.

Ser ikke helt forskjellen, men jeg kan godt endre den, men da skjønner jeg ikke hvorfor vi har snapshot-tester i det hele tatt :woozy_face:

Reply:
Er ikkje farlig. Gjør det du tenker er best.